core/shared/src/test/scala/eu/timepit/refined/BooleanValidateSpec.scala

```diff
@@ -4,18 +4,18 @@ import eu.timepit.refined.TestUtils._
 import eu.timepit.refined.api.Validate
 import eu.timepit.refined.boolean._
 import eu.timepit.refined.char._
-import eu.timepit.refined.numeric.{ Greater, Less }
+import eu.timepit.refined.numeric.{Greater, Less}
 import org.scalacheck.Prop._
 import org.scalacheck.Properties
 import shapeless.nat._
-import shapeless.{ ::, HNil }
+import shapeless.{::, HNil}
 
 class BooleanValidateSpec extends Properties("BooleanValidate") {
 
-  type FF[Op[_, _]] = False Op False
-  type FT[Op[_, _]] = False Op True
-  type TF[Op[_, _]] = True Op False
-  type TT[Op[_, _]] = True Op True
+  type FF [Op[_, _]] = False Op False
+  type FT [Op[_, _]] = False Op True
+  type TF [Op[_, _]] = True Op False
+  type TT [Op[_, _]] = True Op True
```

Comment:
btw, the type signature here doesn't fit on 80 characters

```diff
+  implicit def lowerCaseArbitrary[F[_, _]: RefType]: Arbitrary[F[
+          Char, LowerCase]] = arbitraryRefType(Gen.alphaLowerChar)
```

I'm sure `--maxColumn 100` will yield better results.